_about/notes/0.8.md

```diff
@@ -8,4 +8,55 @@ type: markdown
 ---
 {% include home.html %}
 
-FILL IN BEFORE RELEASE
+In addition to the usual pile of bug fixes and performance improvements, this release includes the new or
+updated features detailed below.
+
+## Networking
+
+- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our new traffic
+management configuration model. The new model adds many new features and addresses usability issues
+with the prior model. There is a conversion tool built into `istioctl` to help migrate your config from
+the old model. [Learn more about the new traffic management model]({{home}}/docs/tasks/traffic-management-v1alpha3/).
+
+- **Envoy V2**. Users can choose to inject Envoy V2 as the sidecar. In this mode, Pilot uses the V2 Envoy API to program the proxies
+in the mesh, which increases effective scalability and should eliminate spurious 404 errors. [TBD: docs on how to control this?]
+
+- **Constrain Inbound Ports**. Restrict the inbound ports in a pod to the ones needed by the apps running inside that pod.
+[TBD: docs on this feature?]
+
+## Security
+
+- **Introducing Citadel**. We've finally given a name to our security component. What was
+formerly known as Istio-Auth or Istio-CA is now called Citadel.
+
+- **Multicluster Support**. We support per-cluster Citadel in multicluster deployments such that all Citadels share the same root cert
+and workloads can authenticate each other across the mesh.
+
+- **Authentication Policy**. We've introduced authentication policy that can be used to configure service-to-service
+authentication (mutual TLS) and end user authentication. This is the recommended way for enabling mutual TLS
+(over the existing config flag and service annotations). [TBD: doc link?]
+
+## Telemetry
+
+- **Self-Reporting**. Mixer and Pilot now produce telemetry that flows through the normal
+Istio telemetry pipeline, just like services in the mesh.
+
+## Setup
+
+- **A la Carte Istio**. Istio has a rich set of features, however you don't need to install or consume them all together. By using
+Helm or `istioctl gen-deploy`, users can install only the features they want. For example, users can install Pilot only and enjoy traffic
+management functionality without dealing with Mixer or Citadel.
+Learn more about [customization through Helm](https://istio.io/docs/setup/kubernetes/helm-install.html#customization-with-helm)
+and about [`istioctl gen-deploy`](https://istio.io/docs/reference/commands/istioctl.html#istioctl%20gen-deploy).
+
+## Mixer adapters
+
+- **CloudWatch**. Mixer can now report metrics to AWS CloudWatch.
+[Learn more]({{home}}/docs/reference/config/adapters/cloudwatch.html)
+
+## Known issues with 0.8
+
+- A gateway with virtual services to a headless service won't work ([Issue #5005](https://github.com/istio/istio/issues/5005)).
+
+- Istio-ingress won't have any route rule support. Users must move to the new Gateway and new route
```

Comment:
I would move this to the networking section and call it a conscious decision.
"We no longer support combining Kubernetes Ingress spec with Istio Route Rules, as it has led to several complex bugs and reliability issues. Istio now supports a platform independent ingress/egress Gateway that works across Kubernetes and CloudFoundry and works seamlessly with the routing rules"

Reply:
Is 0.6/0.7 functionality gone or is this statement (I hope) only applying to v1alpha3 config while the old ingress works as before? It needs to be clear (if people can't upgrade from 0.7 without changes)